src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -17,16 +27,16 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
   protected evalscriptUrl: string | null;
   protected dataProduct: string | null;
 
-  public constructor(
-    instanceId: string | null,
-    layerId: string | null = null,
-    evalscript: string | null = null,
-    evalscriptUrl: string | null = null,
-    dataProduct: string | null = null,
-    title: string | null = null,
-    description: string | null = null,
-  ) {
-    super(title, description);
+  public constructor({
+    instanceId = null,
+    layerId = null,
+    evalscript = null,
+    evalscriptUrl = null,
+    dataProduct = null,
+    title = null,
+    description = null,
+  }: ConstructorParameters) {
+    super({ title, description });
     if (layerId === null && evalscript === null && evalscriptUrl === null && dataProduct === null) {
```

Comment:
What happens if I specify a `layerId`, but not an `instanceId`? Unless I'm missing something, a check for that should be added as well 
` if (!(layerId !== null && linstanceId !== null ) || evalscript !== null || evalscriptUrl !== null && dataProduct !== null))`

Reply:
Agree, good catch, we should fix that. It is however a bit of work, because we should check that for all of the layers - would you agree if we put it in a ticket and solve in a separate PR?

Reply:
Agree, good catch, we should fix that. It is however a bit of work, because we should check that for all of the layers - would you agree if we put it in a ticket and solve in a separate PR?

Reply:
Of course, I see you already created it. Resolving.

Reply:
Ah, nevermind... :) I started working on it and realized you were correct, the rest of the checks seem OK and this is the only change that needs to be done (I did write `if` a bit differently though). Unresolving this, fixed in this PR, please check... :)